spec/models/author_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Author, type: :model do
+  subject(:default) { build(:author) }
```

Comment:
```suggestion
  subject(:author) { build(:author) }
```
Melhor não? Mantém sempre o mais claro possível.